Binary files lua-5.2.4.orig/src/.Makefile.un~ and lua-5.2.4/src/.Makefile.un~ differ
diff -pruN lua-5.2.4.orig/src/Makefile lua-5.2.4/src/Makefile
--- lua-5.2.4.orig/src/Makefile	2015-12-06 00:26:37.000000000 +0000
+++ lua-5.2.4/src/Makefile	2015-12-06 00:33:01.000000000 +0000
@@ -6,9 +6,11 @@
 # Your platform. See PLATS for possible values.
 PLAT= none
 
-CC= gcc
-CFLAGS= -O2 -Wall -DLUA_COMPAT_ALL $(SYSCFLAGS) $(MYCFLAGS)
-LDFLAGS= $(SYSLDFLAGS) $(MYLDFLAGS)
+CC ?= gcc
+CFLAGS ?= -O2 -Wall
+CFLAGS += -fPIC -DLUA_COMPAT_ALL $(SYSCFLAGS) $(MYCFLAGS)
+LDFLAGS ?= 
+LDFLAGS += $(SYSLDFLAGS) $(MYLDFLAGS)
 LIBS= -lm $(SYSLIBS) $(MYLIBS)
 
 AR= ar rcu
